cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,26 @@
-describe('', () => {
-  before(() => {
+/* eslint-disable */
+/// <reference types='cypress' />
+describe('check article flow', () => {
+  beforeEach(() => {
+    cy.task('generateUser').then((userDataString) => {
+      const user = new Object(userDataString)
+      cy.login(user.email, user.username, user.password)
+    })
+    cy.visit('/editor')
+  })
 
-  });
+  it.skip('article creation', () => {
+    cy.task('generateArticleData').then((articleDataString) => {
+      const article = new Object(articleDataString)
+      cy.createArticle(article.title, article.description, article.body)
+    })
+  })
 
-  it('', () => {
-
-  });
-
-  it('', () => {
-
-  });
-});
+  it('article deletion', () => {
+    cy.task('generateArticleData').then((articleDataString) => {
+      const article = new Object(articleDataString)
```

Comment:
The use of `new Object(articleDataString)` is incorrect. If `articleDataString` is a JSON string, it should be parsed using `JSON.parse(articleDataString)` to convert it into a JavaScript object.